package/libyui.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Apr 22 15:26:30 UTC 2021 - Oleksandr Orlov <oorlov@suse.com>
+
+- Ignore ampersand when search item in YDumbTab with rest-api
```

Comment:
Please, add ticket reference to the changelog entry (OBS/IBS do not accept submissions without jira or bugzilla reference). We can reuse bsc#1139747 here.